Assets/MixedRealityToolkit/Utilities/Gltf/Serialization/ConstructGltf.cs

```diff
@@ -246,6 +246,9 @@ private static async Task<Material> CreateMRTKShaderMaterial(GltfObject gltfObje
                 material.mainTexture = gltfObject.images[gltfMaterial.pbrMetallicRoughness.baseColorTexture.index].Texture;
             }
 
+            material.EnableKeyword("_DIRECTIONAL_LIGHT");
```

Comment:
Is this related to this change?

Reply:
Doh, good catch. That should not be there.